sorc/ncep_post.fd/MISCLN.f

```diff
@@ -1206,28 +1207,6 @@ SUBROUTINE MISCLN
             allocate(GTGFD(ISTA:IEND,JSTA:JEND,NFDCTL))
             call FDLVL_MASS(ITYPEFDLVLCTL,NFDCTL,HTFDCTL,GTG,GTGFD)
 !           print *, "GTG 467 Done GTGFD=",me,GTGFD(IM/2,jend,1:NFDCTL)
-
```

Comment:
@EdwardColon-NOAA May I know the intention of removing this gtg related code? Does it have conflict with calculation of  SPC fields?